src/screens/authentication/data/data.tsx

```diff
@@ -16,11 +16,12 @@ export interface NFT {
   time: string;
 }
 const ListIntro: Array<Intro> = [
-  {
-    id: 0,
-    image: images.intro1,
-    title: 'The new NFT marketplace',
-  },
+  // TODO: enable in the next version.
```

Comment:
hi @ThanhNguyen140797 @djhahe , please help me to review again. We need to hide the NFT screen in welcome screen alos.

Reply:
LGTM